static/js/resize.js

```diff
@@ -270,6 +276,10 @@ exports.handler = function () {
     // but before we've loaded in the messages; in that case, don't
     // try to scroll to one.
     if (current_msg_list.selected_id() !== -1) {
+        if (mobile) {
+            popovers.set_suppress_scroll_hide();
+        }
+
```

Comment:
What's the reproducer for the issue this corner case resolves?

Reply:
Are you asking when the resize triggers a scroll? Answer: always, if there’s a selected message.